src/helpers/dateUtils.js

```diff
@@ -51,7 +52,7 @@ export const formatCountDown = (hoursLeft: number) => {
   if (hoursLeft < URGENCY_THRESHOLD) {
     return nmins ? `${nhours}h ${nmins}min` : `${nhours}h`;
   }
-  return `${ndays} days`;
+  return `${ndays} ${timeUnits.days}`;
```

Comment:
Please use standard translate function instead- check the readme file and implementation for booking statuses, use function named `__t` and run `translation:collect`